include/uv.h

```diff
@@ -1453,8 +1453,17 @@ UV_EXTERN void uv_key_set(uv_key_t* key, void* value);
 typedef void (*uv_thread_cb)(void* arg);
 
 UV_EXTERN int uv_thread_create(uv_thread_t* tid, uv_thread_cb entry, void* arg);
+UV_EXTERN int uv_thread_cpumasksize(void);
```

Comment:
In #1389 we're discussing process affinity.  The size of a cpu mask is the same for both process and thread affinities.  I suggest that we rename `uv_thread_cpumasksize` to  `uv_cpumasksize`, `uv_cpumask_size`, or some other name that is not thread-specific.